frostsnapp/lib/wallet.dart

```diff
@@ -727,3 +732,129 @@ Uri getBlockExplorer(BitcoinNetwork network) {
     return Uri.parse("https://mempool.space/signet/");
   }
 }
+
+class BackupWarningBanner extends StatefulWidget {
+  final FrostKey frostKey;
+  final BackupManager backupManager;
+  const BackupWarningBanner({
+    required this.frostKey,
+    required this.backupManager,
+    Key? key,
+  }) : super(key: key);
+  @override
+  State<BackupWarningBanner> createState() => _BackupWarningBannerState();
+}
+
+class _BackupWarningBannerState extends State<BackupWarningBanner>
```

Comment:
I don't understand why this has to be stateful. You have a Stream and you want to conditionally build the widget based on the value of the stream. That's what StreamBuilder is for.